src/phoenix/proto/trace/v1/trace.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
```

Comment:
this is nested under `v1` - are we truly intending to support multiple versions? Because that might be painful to do.